docker/Dockerfile

```diff
@@ -48,7 +52,8 @@ RUN apt-get update && apt-get install -y \
   curl \
   nano
 
-RUN useradd -M -U ticktock
+RUN addgroup --quiet --gid ${GID} ticktock && \
+  adduser --quiet --uid ${UID} --disabled-login --disabled-password --no-create-home --gecos "" --ingroup ticktock ticktock
```

Comment:
Good point.
I have an according container running and tested a `docker exec -it ticktock /bin/bash` just a minute ago - works as expected because `docker exec -it` does neither depend on a home directory nor a login password.